cluster-autoscaler/scale_up.go

```diff
@@ -125,7 +128,17 @@ func ScaleUp(unschedulablePods []*kube_api.Pod, nodes []*kube_api.Node, migConfi
 		if err := gceManager.SetMigSize(bestOption.migConfig, newSize); err != nil {
 			return false, fmt.Errorf("failed to set MIG size: %v", err)
 		}
+
+		for pod := range bestOption.estimator.PodsToSchedule {
+			recorder.Eventf(pod, kube_api.EventTypeNormal, "CausedScaleUp",
+				"pod caused scale-up, mig: %s, sizes (current/new): %d/%d", bestOption.migConfig.Name, currentSize, newSize)
+		}
+
 		return true, nil
 	}
+	for pod := range podsRemainUnshedulable {
+		recorder.Event(pod, kube_api.EventTypeNormal, "DoseNotCauseScaleUp", "pod doesn't cause scale-up (doesn't match scale-up predicate)")
```

Comment:
The message is weird for me. IMO we should say that we tried to help this pod be we wasn't able to do it.